app/src/test/java/com/example/marcinwisniewski/intivetrainingapp/MovieListViewModelTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.marcinwisniewski.intivetrainingapp
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieData
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieRepository
+import com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel.MovieListViewModel
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import io.reactivex.Single
+import org.amshove.kluent.shouldBe
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+const val DEFULT_MOVIE_ID = 1
+const val DEFULT_MOVIE_TITLE = "title"
+const val DEFULT_MOVIE_POSTER = "default/3"
+const val DEFULT_MOVIE_YEAR = "1991"
+const val DEFULT_MOVIE_GENRE = "dramat"
+
+class MovieListViewModelTest {
+
+    @Rule
+    @JvmField
+    var rule = InstantTaskExecutorRule()
+
+    @Rule
+    @JvmField
+    val schedulers = RxImmediateSchedulerRule()
+
+    private lateinit var mockedMovieRepository: MovieRepository
+    private lateinit var testedMovieListViewModel: MovieListViewModel
+
+    @Before
+    fun setUp() {
+        mockedMovieRepository = mock()
+        testedMovieListViewModel = MovieListViewModel(mockedMovieRepository)
+    }
+
+    @Test
+    fun testReceivedEmptyMovieList() {
+        // Given
```

Comment:
I think you don't need to put comment. Empty line between given-when and when-then is enough

Reply:
Fixed